File: benchmark/results/java-invocation-script/run-004/b-baseline/output/com/metaengine/demo/notification/services/DeliveryDispatcher.java
package com.metaengine.demo.notification.services;

import com.metaengine.demo.notification.aggregates.Notification;
import java.util.List;

/** DeliveryDispatcher domain service in the notification domain. */
public class DeliveryDispatcher {
    /** Dispatch a Notification from a partial input. */
    public Notification create(Notification input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a Notification by id, or null if not found. */
    public Notification findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List Notifications up to limit. */
    public List<Notification> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a Notification by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-003/b-baseline/output/metaengine_demo/notification/services/delivery_dispatcher.py
"""DeliveryDispatcher for the notification domain."""
from typing import Optional

from metaengine_demo.notification.aggregates.notification import Notification


class DeliveryDispatcher:
    """DeliveryDispatcher domain service."""

    def create(self, input: Optional[Notification]) -> Notification:
        """Dispatch a new notification."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[Notification]:
        """Look up a dispatched notification by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[Notification]:
        """List dispatched notifications."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """Cancel a dispatched notification."""
        raise NotImplementedError("not implemented")
